post-processor/digitalocean-import/post-processor_test.go

```diff
@@ -9,3 +9,26 @@ import (
 func TestPostProcessor_ImplementsPostProcessor(t *testing.T) {
 	var _ packer.PostProcessor = new(PostProcessor)
 }
+
+func TestPostProcsor_extractImageArtifact(t *testing.T) {
+	tt := []struct {
+		Name      string
+		Source    string
+		Artifacts []string
+	}{
+		{Name: "EmptyArtifacts"},
+		{Name: "SingleArtifact", Source: "Sample.img", Artifacts: []string{"Sample.img"}},
+		{Name: "SupportedArtifact", Source: "Example.tar.xz", Artifacts: []string{"Sample", "SomeZip.zip", "Example.tar.xz"}},
+		{Name: "FirstSupportedArtifact", Source: "SomeVMDK.vmdk", Artifacts: []string{"Sample", "SomeVMDK.vmdk", "Example.xz"}},
+		{Name: "NonSupportedArtifact", Artifacts: []string{"Sample", "SomeZip.zip", "Example.xz"}},
+	}
+
+	for _, tc := range tt {
+		tc := tc
+		source, _ := extractImageArtifact(tc.Artifacts)
```

Comment:
You could also check if it really returns an error here to make sure who's using `extractImageArtifact` will know that `source` doesn't contain the expected value.

Reply:
Good idea. Adding now :+1: